README.md

```diff
@@ -0,0 +1,23 @@
+## Contents Of This File
+
+ * About EzContent_api
+ * Key features
+ * Drupal 8 Installation
+
+## About EzContent_api
+
+This module adds JSON API functionality to EZContent
+(https://www.drupal.org/project/ezcontent) profile. This module extends the 
+JSON API module provided by Drupal core to enabled the following features:
+
+## Key Features
+
+-> Preview unpublished content in de-coupled front-ends.
```

Comment:
Make use of order list, please refer https://github.com/srijanone/ezcontent/blob/8.x/README.md